sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared_access_signature.py

```diff
@@ -316,7 +316,7 @@ def generate_file_sas(
     :param permission:
         The permissions associated with the shared access signature. The
         user is restricted to operations allowed by the permissions.
-        Permissions must be ordered read, write, delete, list.
+        Permissions must be ordered read, add, create, write, delete.
```

Comment:
same here ^_^ lets make it `read, write, delete, list`

Reply:
This is typed as BlobSasPermissions, see comment above.

Reply:
replied above ^_^